### PR TITLE
chore: Fixes Story and Article Content Types

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Fixes stories and articles from Rock to resolve as `UniversalContentItem` type. Before, everything was resolving as `MediaContentItem` because all of our content in Rock has available slots for video.

### How do I test this PR?

Just poll the user feed, you should see all types now.

## TODO

- [x] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings
- [x] Upload Screenshots/GIF(s) if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

---

> The purpose of a PR Review is to _improve the quality of the software._

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -110,14 +110,15 @@ ROCK_MAPPINGS:
       EntityType: ContentChannelItem
     UniversalContentItem:
       EntityType: ContentChannelItem
+      ContentChannelId: [16, 28]
```
Comment (richarddubay) [inline config.yml:113]: This should include news as well. 
```suggestion
      ContentChannelId: [16, 27, 28]
```

**Comment by richarddubay** [comment]
Oh, maybe this fixes the thing with the cards in the search feed I was seeing in that other PR in `corinth`. 

We didn't have problems with the other cards not showing media elements before when articles, stories, and news were returning `DefaultCard`. I think this is probably fine, but we need to make that decision before we worry about changing the types we're returning here.

**Comment by redreceipt** [comment]
I don't understand, so what needs to change here?

**Comment by richarddubay** [comment]
So I think you need to add `news` for starters. That was the code suggestion.

Then I questioned whether we needed this change. We currently have 2 card types in our feeds. With the changes that you're proposing in the PR in `corinth`, you are removing one of those types. I have questioned in that PR whether or not that was something we wanted to do. I don't see the point in making the change suggested here if what we really need to do is to change the client to go back to having 2 card types in the feeds. Does that make sense?

**Comment by redreceipt** [comment]
This PR has nothing to card types, it simply says “news and articles should be UniversalContentItem” we should worry about how that looks client side. Unless you believe everything should be MediaContentItem as it currently is. In which case I’d love to know your thoughts.

—
Michael

**Comment by richarddubay** [comment]
Yeah, okay. The more I thought about it, I guess this doesn't matter that much for the client side conversation. It's probably right that we actually define our content channels as content types in the config instead of leaving it to chance.

**Pull request merged**